plugins/signalmonitor/signalhistorydelegate.cpp

```diff
@@ -47,10 +47,10 @@ SignalHistoryDelegate::SignalHistoryDelegate(QObject *parent)
     , m_visibleOffset(0)
     , m_visibleInterval(15000)
     , m_totalInterval(0)
+    , m_fps(0)
 {
     connect(m_updateTimer, SIGNAL(timeout()), this, SLOT(onUpdateTimeout()));
-    m_updateTimer->start(1000 / 25);
-    onUpdateTimeout();
+    setFPS(15);
```

Comment:
Isn't this addressing the problem on the wrong side? Shouldn't we rather rate-limit the dataChanged signals in the signal history model already, like we do in the meta object model? Maybe that rate limiter should even move to RemoteModelServer generically? Performance on the client isn't really out problem after all, the problem is poor performance on the target, excessive client operations are usually merely a symptom of that.

Reply:
Yes, would you prefer it to be removed ?